I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


File "game/limbo.rpy", line 17: end of line expected.
    with dissolve with Pause(1)
                  ^

Ren'Py Version: Ren'Py 7.3.5.606
Wed May 13 11:07:22 2020
